servicebroker/test/routes/binding_test.js

```diff
@@ -60,11 +61,14 @@ describe('binding RESTful API', function() {
   };
   var policy = { "policy": "testPolicy" };
   before(function(done) {
-    server = require(path.join(__dirname, '../../lib/index.js'));
+    server = require(path.join(__dirname, '../../lib/server.js'))(configFilePath);
```

Comment:
We should just require the server outside, and just call the function directly each time.
We don't need the delete below either.